synapse/config/ratelimiting.py

```diff
@@ -136,6 +136,11 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
             defaults={"per_second": 0.003, "burst_count": 5},
         )
 
+        self.rc_invites_per_issuer = RateLimitConfig(
```

Comment:
@Yoric, @reivilibre: please do remember to add new config settings to the documentation, otherwise people get [angry](https://github.com/matrix-org/synapse/issues/8159#issuecomment-1189303708).

Reply:
see https://github.com/matrix-org/synapse/issues/13330